README.md

```diff
@@ -80,3 +80,13 @@ If you get an `ImportError` from python, you may need to set the `PYTHONPATH` en
 ```bash
 export PYTHONPATH=/path/to/venv/lib/python3.x/site-packages
 ```
+
+When submitting a PR we check coverage. You can check coverage locally with
+
+```bash
+$ cargo llvm-cov --workspace
+# generate html
+$ cargo llvm-cov --html
+# open it
+$ open -a <your_browser> target/llvm-cov/html/index.html
+```
```

Comment:
I think it makes sense to explain that we measure coverage from both Rust and Python tests:

````suggestion
```

Note this only checks the coverage generated by the Rust tests. We also measure coverage generated by the Python tests in CI, which means missing coverage locally isn't always an issue.
````